modules/product/src/test/java/com/opengamma/strata/product/bond/ResolvedCapitalIndexedBondTest.java

```diff
@@ -55,7 +56,7 @@
       .gearing(ValueSchedule.of(COUPON))
       .index(US_CPI_U)
       .lag(Period.ofMonths(3))
-      .interpolated(true)
+      .indexCalculationMethod(INTERPOLATED)
       .firstIndexValue(198.475)
       .build();
   private static final double NOTIONAL = 10_000_000d;
```

Comment:
Should add a test on the special accrual method for JAPAN_IL (line 376 in ResolvedCapitalIndexedBond). One test including 29 February is required.